README.md

```diff
@@ -2,57 +2,48 @@
 
 ## The Task:
 
-Welcome to On's Fullstack Engineer coding challenge! To help us assess your coding skills, we'd like you to build a small application with a **Ruby backend and a Javascript framework on the frontend**.
+Welcome to On's Fullstack Engineer coding challenge! To help us assess your coding skills, we'd like you to build a small application with a **Ruby backend and a Javascript framework on the frontend**. The app should fetch retailer data from an API, store it locally, and display the retailers on a map.
 
-We will provide you access to the Salesforce API with data related to On dealers. The app should:
-  1. Synchronize all relevant dealers from Salesforce into its own database
-   - Relevant dealers are all accounts with the field `E_Shop_Dealer__c == "Dealer and Point of Sale"`
-   - Consider the case that dealers can be deleted from Salesforce
-  2. Display all dealers from the local database in a list and on a map
-   - The map and list should be linked: clicking on a dealer in the map should highlight it in the list and vice versa
-   - The page should look acceptable on different devices
+Breaking down the task the app should:
 
-## Using the Salesforce API
+1. Synchronize all records from a remote system via the provided API into its own database
 
-Salesforce provides an API which you will use to retrieve the dealers.
+- Consider the case that records can be deleted in the remote system
 
-For an introduction to the API: click [here](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_what_is_rest_api.htm)
+2. Display all dealers from the local database in a list and on a map
```

Comment:
Dealers -> companies

Reply:
Nvm, I see you mentioned that company simulates a dealer. Probably enough.

Reply:
I would actually stick with either `retailer` or `dealer` throughout the whole task description. Currently it's a mix of both.